grunt/compress.js

```diff
@@ -19,5 +19,26 @@ module.exports = {
                 dest: '/'
             }
         ]
+    },
+    documentation: {
+        options: {
+            archive: path.join(process.cwd(), 'target', 'docs', 'ux-aspects-docs-<%= package.version %>.tgz'),
+            mode: 'tgz'
+        },
+        files: [
+            {
+                cwd: path.join(process.cwd()),
+                src: [
```

Comment:
Shouldn't this be `src: 'dist/docs'`?

Reply:
So just to clarify, is this supposed to contain the site that will be published to gh-pages? or contain the contents of the ux-aspects-docs package? I wasn't sure which it was supposed to be

Reply:
Yes, it's the documentation site from dist/docs.

Reply:
Great thanks! The changes should now be made!